google-cloud-tools-plugin/resources/META-INF/plugin.xml

```diff
@@ -164,8 +164,9 @@
                      key="unique.rest.signature.name" groupKey="inspections.group.name" enabledByDefault="true"
                      level="ERROR" implementationClass="com.google.cloud.tools.intellij.appengine.validation.RestSignatureInspection"/>
 
-    <postStartupActivity implementation="com.google.cloud.tools.intellij.PluginCompatibilityCheck"/>
-    <postStartupActivity implementation="com.google.cloud.tools.intellij.ConflictingAppEnginePluginCheck"/>
+    <postStartupActivity implementation="com.google.cloud.tools.intellij.startup.PluginCompatibilityCheck"/>
```

Comment:
It appears that postStartupActivity may actually be the right choice here. The issue with using project or module components is that this feature requires us to inspect the project structure (currently using `PSIManager` via the `DefaultAppEngineAssetProvider` which we use to read the module's `appengine-web.xml`).  PSI files appear not to be initialized until later in the project initialization lifecycle (even after the `projectOpened` hooks in both `ModuleComponent` and `ProjectComponent`), which results in an exception when we try to access PSI files via either of these components.

I'm going to revert to the prior implementation using postStartupActivity.